modules/arm_plugin/src/transformations/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_arm_cpu_transformations")
+
+file(GLOB_RECURSE SOURCES_TRANSFORMATIONS
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+
+add_library(${TARGET_NAME} STATIC ${SOURCES_TRANSFORMATIONS})
+
+ie_arm_neon_optimization_flags(neon_flags)
+set_source_files_properties(${SOURCES_TRANSFORMATIONS} PROPERTIES COMPILE_OPTIONS "${neon_flags}")
```

Comment:
I suppose NEON is not needed for transformations as well

Reply:
@ilya-lavrenov done